troposphere/static/js/components/images/detail/description/EditDescriptionView.react.js

```diff
@@ -20,7 +20,7 @@ export default React.createClass({
     render: function() {
         return (
         <div className={this.props.className}>
-            <h4 className={this.props.titleClassName}>{this.props.title}</h4>
+            <h4 className={"t-body-2 " + this.props.titleClassName}>{this.props.title}</h4>
```

Comment:
This could be updated to:
```
render: function() {
let titleClass = this.props.titleClassName;
...
<h4 className={`t-body-2 ${titleClass}`}>
...
```